src/components/FoodForm.js

```diff
@@ -1,71 +1,151 @@
 import {nanoid} from 'nanoid';
 import {useState} from 'react';
+import styled from 'styled-components';
 
+import {StyledResults} from './styled/StyledContainer';
 import {StyledDoneLabel} from './styled/StyledDoneLabel';
 import {StyledForm} from './styled/StyledForm';
+import {StyledImageCenter} from './styled/StyledImage';
 import {StyledInput} from './styled/StyledInput';
+import {StyledAtag} from './styled/StyledLink';
 import {StlyedList} from './styled/StyledList';
-import {StyledListItems} from './styled/StyledListItems';
+import {StyledListItems, StyledFetchItems} from './styled/StyledListItems';
+import {StyledBlackButton, StyledWhiteButton} from './styled/StyledNavButton';
+import {StyledCalories} from './styled/StyledNew';
+
+const StyledFetchButtons = styled.form`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: 6px;
+`;
+
+const StyledSearch = styled.div`
+	display: flex;
+	justify-content: center;
+	margin: 16px;
+	padding: 16px;
+	border-radius: 16px;
+	background: rgba(0, 0, 0, 0.5);
+	color: white;
+`;
+
+const StyledNutrientList = styled.ul`
+	list-style: none;
+`;
+
+const StyledText = styled.p`
```

Comment:
Why do you need an extra p element inside your `StyledFetchItems`? You could style them directly.